catalogue/infrastructure/terraform/modules/service/main.tf

```diff
@@ -0,0 +1,28 @@
+module "task" {
+  source = "github.com/wellcomecollection/terraform-aws-ecs-service.git//task_definition/container_with_sidecar?ref=v1.1.1"
+
+  task_name = var.namespace
+
+  cpu    = var.app_cpu + var.nginx_cpu
+  memory = var.app_memory + var.nginx_memory
```

Comment:
💯 to `+`!

Reply:
Do you need to take docker using up 4mb or something of memory?

Reply:
Not sure I follow? We need to give nginx some resources (and the task def needs that for all containers in the task def and for the whole thing - for some reason).